honeybee_radiance_folder/folder.py

```diff
@@ -500,6 +501,56 @@ def dynamic_scene(self, indoor=False, reload=False):
         return self._dynamic_scene_indoor if indoor \
             else self._dynamic_scene
 
+    def grid_info(self, info_type_is_model=False):
+        """Parse the appropriate grids_info file to return high level information about
+        the sensor grids contained in the model folder.
+
+        Args:
+            info_type_is_model: If set to True, the _model_grids_info.json, which is
+            written out after simulations will be parsed. Else the _info.json will be
+            parsed. Defaults to False as the _info.json file is always expected to be
+            present.
+        Returns:
+            The list containing information about sensor grids that are written to the
+            folder after model export.
+        """
```

Comment:
You're missing the extra line and white spaces.

```suggestion
        Args:
            info_type_is_model: If set to True, the _model_grids_info.json, which is
                written out after simulations will be parsed. Else the _info.json will be
                parsed. Defaults to False as the _info.json file is always expected to be
                present.

        Returns:
            The list containing information about sensor grids that are written to the
            folder after model export.
        """
```

Reply:
This is the case for most of the descriptions.  This way we make sure it looks good in the docs.